packages/ag-charts-types/src/chart/localeOptions.ts

```diff
@@ -1,8 +1,10 @@
+import type { Formatter } from './callbackOptions';
+
 export interface AgLocaleThemeableOptions {
     /** A record of locale texts keyed by id. */
     localeText?: Record<string, string>;
     /** Formatter that generates the text displayed to the user. */
-    getLocaleText?: (params: MessageFormatterParams) => string | undefined;
+    getLocaleText?: MessageFormatter;
```

Comment:
Do the docs still look the same after this?

Reply:
Yes, it should. the function resolving was fixed to support it.